airbyte-integrations/connectors/source-stripe/source_stripe/manifest.yaml

```diff
@@ -427,6 +859,555 @@ definitions:
             request_parameters:
               types[]: '{{["radar.early_fraud_warning.created", "radar.early_fraud_warning.updated"]}}'
 
+    customer_balance_transactions:
+      $ref: "#/definitions/entity_stream"
+      $parameters:
+        name: customer_balance_transactions
+      retriever:
+        $ref: "#/definitions/base_retriever"
+        $parameters:
+          path: customers/{{ stream_partition.customer_id }}/balance_transactions
+        partition_router:
+          type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - type: ParentStreamConfig
+              parent_key: id
+              partition_field: customer_id
+              stream:
+                $ref: "#/definitions/streams/customers"
+              incremental_dependency: true
+      incremental_sync:
+        $ref: "#/definitions/entity_single_slice_cursor"
+        cursor_field: created
+        global_substream_cursor: true
```

Comment:
Enabling global state makes this stream fallback on the non-concurrent engine because of [this condition](https://github.com/airbytehq/airbyte-python-cdk/blob/24cbc51774274b6dfef0fe08544d12fe31e2bb2e/airbyte_cdk/sources/declarative/concurrent_declarative_source.py#L363-L365). Should we update this condition to `and isinstance(declarative_stream.retriever.stream_slicer, (GlobalSubstreamCursor, PerPartitionWithGlobalCursor))`?

Reply:
will be added in https://github.com/airbytehq/airbyte-python-cdk/pull/490